docs/DeveloperGuide.md

```diff
@@ -512,10 +512,10 @@ Figure X. Workflow of an AddAppointment command with the help of CalendarManager
 
 #### 4.4.2. Current Implementation
 The `CalendarManager` class in the `Model` component contains an `AppointmentCalendar` object, storing the currently set 
-`year`, `month` and `day`. Note that the `year`, `month` and `day` attributes may not necessarily be storing the 
+year, month and day. Note that the year, month and day attributes may not necessarily be storing the 
```

Comment:
Should we leave the ` but change it to `Year`, `Month` and `Day` instead?

Reply:
^ agreed

Reply:
@jianhandev was saying might look ugly (which is true) but maybe for consistent styling?